README.rst

```diff
@@ -614,27 +614,45 @@ Install the system dependencies:
     sudo apt install -y sqlite3 libsqlite3-dev openssl libssl-dev
     sudo apt install -y gdal-bin libproj-dev libgeos-dev libspatialite-dev libsqlite3-mod-spatialite
 
+Fork and clone the forked repository:
+
+.. code-block:: shell
+
+    git clone git://github.com/<your_fork>/openwisp-controller
+
+Navigate into the cloned repository:
+
+.. code-block:: shell
+
+    cd openwisp-controller/
+
 Launch Redis:
 
 .. code-block:: shell
 
     docker-compose up -d redis
 
-Install your forked repo:
+Setup and activate a virtual-environment. (we'll be using  `virtualenv <https://pypi.org/project/virtualenv/>`_)
 
 .. code-block:: shell
 
-    git clone git://github.com/<your_fork>/openwisp-controller
-    cd openwisp-controller/
-    python setup.py develop
+    python -m virtualenv env
+    source env/bin/activate
+
+Make sure that you are using pip version 20.2.4 before moving to the next step:
+
+.. code-block:: shell
+
+    pip install -U pip==20.2.4
```

Comment:
Let's also throw in `wheel` and `setuptools` here. This will make the documentation align with what we do in CI. 

P.S.: My new virtual environments never have `wheel` installed by default and pip throw a bunch of errors during the installation that it failed to build wheel for some module. These errors might freak out new contributors who are not versed with Python.

Reply:
I have made the change.